src/ray/raylet/scheduling/local_resource_manager.cc

```diff
@@ -501,11 +501,20 @@ ray::gcs::NodeResourceInfoAccessor::ResourceMap LocalResourceManager::GetResourc
 }
 
 void LocalResourceManager::OnResourceChanged() {
```

Comment:
Hm should we rename this API? NotifyResourceChangedAndShutdownIfNeeded?

Reply:
OnResourceOrStateChanged